### PR TITLE
New Drug Updates

### PR DESCRIPTION
**Pull request by alival000**
new drug model added to the form and servlet

### CODE REVIEW

**Review by EdithBenvenuto — APPROVED** [review]
LGTM

File: src/main/java/com/google/sps/servlets/NewDrugServlet.java [hunk 1]
```diff
@@ -43,18 +43,31 @@
...
+
 
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
     KeyFactory keyFactory = datastore.newKeyFactory().setKind("Drug");
     FullEntity drugEntity = 
         Entity.newBuilder(keyFactory.newKey())
             .set("Name", name)
+            .set("Classification", classification)
             .set("Aliases", aliases)
+            .set("Safety Description", safetyDescription)
```
Comment (mcollins42) [inline src/main/java/com/google/sps/servlets/NewDrugServlet.java]: I think this works, but I would avoid spaces in property names. I would probably just skip the spaces and use SafetyDescription. The same for the remaining properties.

**Review by mcollins42 — APPROVED** [review]
Thanks for putting this together! Just the one comment on naming.

File: src/main/java/com/google/sps/servlets/NewDrugServlet.java (hunk 1 above)
Comment (alival000) [reply src/main/java/com/google/sps/servlets/NewDrugServlet.java]: I got rid of the spacing. Do I make a new PR for these changes?

Comment (mcollins42) [reply src/main/java/com/google/sps/servlets/NewDrugServlet.java]: No, you can add the commit to this branch and push to update this PR.

**Pull request merged**